### ISSUE TITLE
Add/modify timer: datetimepicker: sliders of hour and minute should react on mouse wheel

### ISSUE THREAD

**Issue by Micha10** [E2OpenPlugins/e2openplugin-OpenWebif#1444]
The dateTimePicker has the hour and minutes represented as sliders.
You have to grab the slider with the mouse and drag it.

Wish: just hover with your mouse over the slider and change the value by mouse wheel.

**Comment by wedebe** [comment]
There's a plan to completely redesign the add/edit timer functionality in the modern interface to match what happens in AutoTimers. @Micha10 , could you confirm whether you're referring to the modern interface, and if so, whether the AutoTimers add/edit hours/minutes works as you'd like/expect?

**Comment by Micha10** [comment]
Yes, the input field with sliders are working in the modern design.
(by the way. I prefer the old design, because you need less clicks than with the modern one (Add Timers vs (...-menu)/add a timer. 
The remote is always visible vs. has to be activated manually... and more.
